blocks/columns/columns.css

```diff
@@ -37,37 +54,89 @@
     margin: var(--spacer-layout-02) auto;
 }
 
+.section-top > div:first-child {
+    margin: 0 auto;
+    padding-top: var(--spacer-layout-04);
+    }
+
+.content-band div:empty {
+    display: none;
+    }
+
+.content-band.icon-list > div > div {
+    margin-top: var(--spacer-element-05);
+    margin-bottom: var(--spacer-element-06);
+    }
+
+.columns.content-band li {
+    list-style-image: url("../icons/arrow-purple.svg");
+    margin-bottom: var(--spacer-element-03);
+    line-height: var(--line-height-160);
+    }
+
+.section.leadership-content h3 {
+    margin: var(--spacer-element-08) auto var(--spacer-element-05) auto;
+    }
+
+.content-band.icon-list.picture-circle h3 {
+    margin: var(--spacer-layout-01) auto;
+    }
+
+.columns img {
+    width: 100%;
+    }
+
+.leadership-content img {
+    border-radius: 15px;
+    }
+
+.section-top-grid > div > div:nth-child(odd) img {
+    border-radius: 90% 0 0;
+    width: 230px;
+    height: 230px;
+    }
+
+.section-top-grid > div > div:nth-child(even) img {
+    border-radius: 0 90% 0 0;
+    width: 98px;
+    height: 95px;
+    max-width: 98px;
+    }
+
+.content-band.icon-list.picture-circle img {
+    border-radius: 50%;
+    height: 158px;
+    width: 158px;
+    }
+
 .content-band.icon-list h4 {
     font-size: var(--font-size-18);
     line-height: var(--line-height-160);
     letter-spacing: var(--letter-spacing-1);
-    margin: var(--spacer-layout-02) auto;
+    margin: var(--spacer-layout-01) auto;
 }
 
-.columns > div > div > *:first-child {
-    margin-top: 0;
-}
+.content-band.icon-list .only-picture {
+    margin: var(--spacer-element-03) auto var(--spacer-layout-01) auto;
+    }
 
 .columns > div > div > h4:first-child {
     text-transform: uppercase;
 }
 
-.content-band p, .content-band li {
-    font-family: var(--sans-serif-font-light);
-    font-size: var(--font-size-16);
-    font-weight: var(--font-weight-light);
-    line-height: var(--line-height-160);
-    letter-spacing: var(--letter-spacing-1);
-}
+.leadership-content picture {
+    background-color: #e6e3d9;
```

Comment:
This is a unique color only used by leadsership content?

Reply:
Yes, it is the color that is in the background of the images. If the pictures take time to load, the background color will be a visual placeholder.